### PR TITLE
add risk management document

### PR DESCRIPTION
**Pull request by kyriediculous**

### CODE REVIEW

File: resources/risk-management.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    1. Is the node operator an active community member?
+    2. Does it provide value-added services for the community?
+    3. Does it participate in governance proposals, community discussions, and/ or meetups?
+    
+
+Leveraging each of these four components informs our overall view and permits us to choose node operators with strong reputations that maintain highly performative infrastructure and offer competitive returns to Tenderize stakers.
+
+### Important Metrics
+
+To track these criteria and give statistical insight into performance we have a come up with a concise set of KPI that prioritises profitability and reliability.
```
Comment (vmaark) [inline resources/risk-management.md]: typo: `we have a come up with`

File: resources/risk-management.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+1. **Commission Rate:** The rate a node operator charges their Delegators. Delegators claim rewards and are charged a small, percentage-based fee. This helps node operators pay for operational costs, maintenance costs, and more. We analyze the average rates in the market to evaluate the commision rate being applied.
+2. **Track Record**: Of the nodes that qualify based on commission rates, we evaluate its past performance, how long it’s been active, how often it changes its commission rates, whether or not it has been slashed, and how much network fees it has earned.
+3. **Network Dominance:** We believe it is important to make choices which support decentralization. That is why we choose node operators that have low market share in the network relative to the work they produce. This has two benefits: decentralization and a larger reward share for our delegators.
+4. **Community Involvement:** To measure this, we ask ourselves the following questions:
+    1. Is the node operator an active community member?
+    2. Does it provide value-added services for the community?
+    3. Does it participate in governance proposals, community discussions, and/ or meetups?
```
Comment (vmaark) [inline resources/risk-management.md]: typo: `and/ or` should be `and / or`

File: resources/risk-management.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Leveraging each of these four components informs our overall view and permits us to choose node operators with strong reputations that maintain highly performative infrastructure and offer competitive returns to Tenderize stakers.
+
+### Important Metrics
+
+To track these criteria and give statistical insight into performance we have a come up with a concise set of KPI that prioritises profitability and reliability.
+
+- APY% : Gives insight into the compounding profitability
+- 30/90 day fee revenue: Insight into active network participation and important for profitability
+- earned fees to total stake ratio: Checks if the node operators is over-performing or underperforming versus its stake, which is a very useful metric since work allocation is often stake-weighted
+- % fees shared to delegators : Fee commission rate
```
Comment (vmaark) [inline resources/risk-management.md]: typo: `delegators : Fee` -> `delegators: Fee`

File: resources/risk-management.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+- % inflation rewards shared to delegators: Staking rewards commission rate
+- slashing frequency (if applicable): We want node operators that get slashes as frequently as possible, preferably not at all.
```
Comment (vmaark) [inline resources/risk-management.md]: rather `We want node operators to get slashes as rarely as possible`, no?

File: resources/risk-management.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+**What are the risks of being slashed if I stake MATIC tokens with Tenderize?**
+
+- On Polygon, there is the risk of slashing for all delegators and node operators as a design choice. Since network security is in function of a ⅔ +1 honest majority, delegators carry responsibility for their delegations. However as of today, no slashing events have occured in the Polygon network.
+
+**What are the risks of being slashed if I stake AUDIO tokens with Tenderize?**
+
+- There is a possibility of being slashed but as of to date there have been no such events in the Audius network. Slashing can occur by committee and penalties are decided on a case-by-case basis.
+
+**If Tenderize Liquidity Pools have insufficient liquidity to exchange tASSET for ASSET, w what alternatives exist for me to redeem my tASSET for the staked asset on a 1:1 exchange rate?**
```
Comment (vmaark) [inline resources/risk-management.md]: typo: `w what`

**Review by vmaark — CHANGES_REQUESTED** [review]
some small typos and an inverted adjective

**Comment by kyriediculous** [comment]
Thanks @vmaark fixed the typos

**Review by vmaark — APPROVED** [review]
🚀

**Pull request merged**